### ISSUE TITLE
Getting Browser Errors - TypeError, ReferenceError

### ISSUE THREAD

**Issue by SoShoTech** [qooob/authentic-theme#631]
Debian 8
Windows 10, latest Chrome
I'm just logging into virtualmin as usual, and I sporadically get some browser errors which keep the interface from loading - get a continuous loading screen.  I have to empty cache and hard reload to get back to the interface.

Browser output below:

parent.min.js?1831:6 Uncaught TypeError: lang(...).toUpperCase is not a function
    at get_right_panel_sysinfo_data (parent.min.js?1831:6)
    at __si__bg_upd_exec (parent.min.js?1831:6)
    at parent.min.js?1831:6
get_right_panel_sysinfo_data @ parent.min.js?1831:6
__si__bg_upd_exec @ parent.min.js?1831:6
(anonymous) @ parent.min.js?1831:6
init.min.js?1831:6 Uncaught ReferenceError: __mss is not defined
    at HTMLIFrameElement.<anonymous> (init.min.js?1831:6)
    at HTMLIFrameElement.dispatch (bundle.min.js?1831:2)
    at HTMLIFrameElement.r.handle (bundle.min.js?1831:2)
(anonymous) @ init.min.js?1831:6
dispatch @ bundle.min.js?1831:2
r.handle @ bundle.min.js?1831:2

**Comment by qooob** [comment]
Hi,

It seems your cache isn't cleared properly.

I try to open console and then hold reload button and then choose 3 option from the dropdown.

**Comment by qooob** [comment]
I have made a small patch that will be released with 18.32, which should stop this issue.

In case it happens again, I would recommend going through installed _Chrome's Extensions_ and trying to disabling all of them, and then enabling one by one to figure out which one is causing an issue.

I couldn't exactly reproduce an issue. What I fixed is closely related and might affect your bug in positives way.
